lib/galaxy/webapps/galaxy/api/tours.py

```diff
@@ -38,17 +38,17 @@ class FastAPITours:
     registry: ToursRegistry = Depends(get_tours_registry)
 
     @router.get('/api/tours')
-    def index(self) -> TourList:
+    async def index(self) -> TourList:
         """Return list of available tours."""
         return self.registry.get_tours()
 
     @router.get('/api/tours/{tour_id}')
-    def show(self, tour_id: str) -> TourDetails:
+    async def show(self, tour_id: str) -> TourDetails:
```

Comment:
Doesn't this do synchronous I/O?

```python
            with open(tour_path) as f:
                tour = yaml.safe_load(f)
                load_tour_steps(tour)
```

I thought we were supposed to drop these asyncs.

Reply:
I missed this, thank you! I made the change to async for config *and* tours, *then* we had that discussion and I changed it back - but only for config! Correcting now...